dbt_project.yml

```diff
@@ -0,0 +1,47 @@
+
+name: 'snowplow_fractribution'
+version: '1.0.0'
```

Comment:
For the first release we normally go for a minor version and increment them with each feature (let's say when we add a new adapter support). I'd suggest to start with v 0.1.0. We also have a [guide](https://github.com/snowplow-incubator/engineering-resources/blob/master/release-process.md) on the release process that we try to keep, although some things we are more relaxed about, I will try to write down each step that are relevant. 

Just so we follow the usual process, could you please create a release branch (release/snowplow-fractribution/0.1.0) and change this pr request to be merged with that first, instead of main, please? Once it is approved, you can then squash the commits together into one commit during the merge to the release branch (if not allowed for some reason you can perhaps rebase them into one commit and force push to this branch or do the same on the release branch, too) and for the commit message just say something along the lines of `Add fractribution model for Snowflake.` After this one commit is merged to the release branch, a `Prepare for release` commit is added with an entry in the CHANGELOG to specify the details. See an example of how this works [here](https://github.com/snowplow/dbt-snowplow-web/pull/3/commits). There are a couple of more steps, it is perhaps easier to just copy paste a step by step guide:

1. Create a commit on the `release/snowplow-X/Y.Z` branch with message `Prepare for release` with an update to:
    1. The `CHANGELOG` providing the information that should roughly match the commit messages into the branch (including issues closed). Add `(Thanks to @gh_contributor)`  if the commit comes from an external contributor 
    2. The `README.md` to update the version in the adaptor support
    3. The `dbt_project.yml` to update the version
    4. Any custom example or integration test project yaml files (usually version number)
    5. Update the docs to ensure they match the latest version of the branch (run on redshift where possible) 
        
        ```bash
        dbt docs generate
        rm -f docs/catalog.json
        rm -f docs/manifest.json
        rm -f docs/run_results.json
        cp target/catalog.json docs/catalog.json
        cp target/manifest.json docs/manifest.json
        cp target/run_results.json docs/run_results.json
        ``` 
2. Create a merge request from the release branch to the main/master branch
3. Once reviewed, merge into the main/master branch
4. Checkout the branch locally and tag it with the release number:
    1. `git checkout main`
    2.  `git pull`
    3. `get tag 0.x.y`
    4. `git push --tags`
5. Create a Github Release from this tag with:
    1. **Summary**: A brief overview of the release
    2. **Features**: A list of features, should roughyly match the commit messages
    3. **Upgrading**: Information about how to upgrade to the latest version and/or take advantage of new features
6. Merge the relevant PR in the [documentation site](https://github.com/snowplow/documentation) if not already done
7. Delete all branches created as part of the release
8. If this is the first release of a new dbt package, it also needs to be added to the dbt hub. Raise a Pull Request in [https://github.com/dbt-labs/hubcap/ ](https://github.com/dbt-labs/hubcap/)with changes to the hub.json file.
Sorry for this long post but was not sure how different your processes to this so best to keep it verbose. Please let me know if you need more details, not sure how different your wow to this one. Thanks!

Reply:
Thanks Agnes! I haven't really done this before, so the long post is very helpful! I have made the branch and pointed this PR to that branch. From what I understand, I perform the rest of these steps after this PR has been merged to that branch?

Reply:
Yes, exactly! This is an example PR for the dat-labs repo as well, if it helps: https://github.com/dbt-labs/hubcap/pull/115/commits/502bd140fa4de78453c238434f22c052b7d138b0.